plugins/services/src/js/components/forms/EnvironmentFormSection.js

```diff
@@ -231,7 +186,33 @@ class EnvironmentFormSection extends Component {
         <Trans render="p">
           Set up environment variables for each instance your service launches.
         </Trans>
-        {this.getEnvironmentLines(data.env)}
+        {environmentLines.length > 0 ? (
```

Comment:
These look like the changes from [Pierre's PR](https://github.com/dcos/dcos-ui/pull/3828). Did you intentionally include these in your PR?

Reply:
i'm sorry. that's been me assuming that the form colon fix will merge timely. if it's not merged by tomorrow, i'll rebase those changes out of this PR.